stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -97,13 +101,29 @@ public struct Unsafe${Mutable}BufferPointer<Element> {
   ///
   /// - Parameter other: The mutable buffer pointer to convert.
   @inlinable // unsafe-performance
+  @_preInverseGenerics
   public init(_ other: UnsafeMutableBufferPointer<Element>) {
     _position = UnsafePointer<Element>(other._position)
     count = other.count
   }
 % end
 }
 
+@available(*, unavailable)
+extension Unsafe${Mutable}BufferPointer: Sendable {}
```

Comment:
Do we need `where Pointee: ~Copyable` on this? Will the compiler still complain about sendability if say we used `UnsafeBufferPointer<Mutex<Int>>` (dumb example) or would it allow it silently...?

Reply:
This should work as is, but the hidden condition is silly, so I spelled out its suppression elsewhere, and I think we should do the same here.